src/tests/sqlsmith/src/bin/main.rs

```diff
@@ -224,6 +224,12 @@ async fn main() {
     let (tables, mviews, setup_sql) = create_tables(&mut rng, &opt, &client).await;
 
     // Test batch
+    // Queries we generate are complex, can cause overflow in
+    // local execution mode.
+    client
+        .query("SET query_mode TO distributed;", &[])
+        .await
```

Comment:
IIRC this is default option? Where local mode is used?

Reply:
Local mode has been set to distributed mode recently, see comment: https://github.com/singularity-data/risingwave/issues/4807#issuecomment-1223608299